apps/site/src/components/pages/wordpress/availability.tsx

```diff
@@ -0,0 +1,136 @@
+import { Box, Container, List, ListItem, Typography } from "@mui/material";
+
+import { RoadIcon } from "../../icons";
+import { BpWpIcons } from "./bp-wp-icons";
+import { EarlyAccessCTA } from "./early-access-cta";
+
+export const Availability = () => {
+  return (
+    <Container
+      sx={{
+        mb: 10,
+        maxWidth: { xs: "95%", md: "75%", lg: "60%" },
+        fontFamily: "Inter",
+      }}
+    >
+      <Typography
+        sx={{ fontSize: "2rem", lineHeight: 1, fontWeight: 900, mb: 1 }}
+      >
+        General Availability
+      </Typography>
+
+      <Typography
+        component="div"
+        sx={{
+          fontSize: "2rem",
+          lineHeight: 1,
+          color: ({ palette }) => palette.gray[50],
+        }}
+      >
+        The <strong>0.3 spec</strong> and{" "}
+        <Box sx={{ display: "inline-block", whiteSpace: "nowrap" }}>
+          <BpWpIcons />
+        </Box>{" "}
+        <strong>plugin</strong> will ship in February 2023
+      </Typography>
+
+      <Box
+        sx={({ breakpoints }) => ({
+          display: "flex",
+          pt: 4,
+          alignItems: "center",
+          gap: 7.25,
+          [breakpoints.down("lg")]: {
+            gap: 3,
+            flexDirection: "column",
+          },
+        })}
+      >
+        <Box flex={1}>
+          <Typography sx={{ lineHeight: 1.2, maxWidth: "unset", pb: 2.25 }}>
+            The <strong>Þ WordPress plugin</strong> will be launched alongside{" "}
+            <strong>version 0.3</strong> of the
+            <strong>Þ</strong> specification, our largest-ever update that
```

Comment:
![Screenshot 2022-12-15 at 15 29 59](https://user-images.githubusercontent.com/37743469/207901161-e04ce045-06c7-4bd6-b801-d678284bc4c2.png)

```suggestion
            <strong> Þ</strong> specification, our largest-ever update that
```